src/lib/network/edit_nic_name.rb

```diff
@@ -63,12 +63,16 @@ def run
           next
         end
 
+        current_rule = LanItems.GetItemUdevRule(LanItems.current)
+
         if UI.QueryWidget(:udev_type, :CurrentButton) == :mac
           rule_key = MAC_UDEV_ATTR
           rule_value = @mac
+          LanItems.RemoveKeyFromUdevRule(current_rule, "ATTR{dev_port}")
         else
           rule_key = BUSID_UDEV_ATTR
           rule_value = @bus_id
+          LanItems.ReplaceItemUdev("ATTR{dev_port}", "ATTR{dev_port}", dev_port(@old_name))
```

Comment:
I added the dev_port?(@old_name) locally and planned to update the PR with the first CR, I was also thinking in use directly the method and maybe move it to some Udev class but thinking to do it as part of the documentation PBI.